cves/kernel/CVE-2016-5400.yml

```diff
@@ -386,12 +409,14 @@ lessons:
 
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
-  defense_in_depth:
-    applies:
-    note:
-  least_privilege:
+  defense_in_depth: 
     applies:
     note:
+  least_privilege: 
+    applies: true
+    note: |
+      should keep track of the bounds of the memory you're changing
```

Comment:
The fix adds proper error handling and cleanup steps to prevent a memory leak, contributing to the overall defense-in-depth strategy.

Reply:
^ this applies for defense in depth, not sure why it highlighted least privilege